x/ccv/consumer/keeper/relay.go

```diff
@@ -196,13 +196,17 @@ func (k Keeper) SendPackets(ctx sdk.Context) {
 		)
 		if err != nil {
 			if clienttypes.ErrClientNotActive.Is(err) {
```

Comment:
The check for `if clienttypes.ErrClientNotActive.Is(err)` is no longer needed since behavior is the same whether that statement is true or not. The type of error will be printed